docs/getting-started-guides/clc.md

```diff
@@ -70,7 +80,7 @@ guide to install the requirements and install the script.
 ```shell
   # system
   apt-get update
-  apt-get install -y git python python-crypto
+  apt-get install -y curl git python python-dev python-crypto
   curl -O https://bootstrap.pypa.io/get-pip.py
   python get-pip.py
 
```

Comment:
Line 88 you probably don't mean to create a home user directory. `mkdir -p ~home` would create /home/home/ I'm assuming you mean ~/k8s-on-clc but I don't think that's necessary at all since `git clone ...` will create an adm-kubernetes-on-clc directory.